Week-6/W6Exercise6.3/Index.js

```diff
@@ -0,0 +1,25 @@
+const sortArr = (arr) => {
+    let low = 0, high = arr.length - 1, mid = 0, temp = 0;
+    while (mid <= high) {
```

Comment:
Explanation for this loop?

Reply:
Sorting 0, 1, 2

- 4 variables are initialized, mid for 1, low for 0 and high for 2, and temp for swapping values
- 3 conditions checks to check if the current element is 0, 1 or 2
- If element is 0, then swapping element to the starting side of the array, where low variables of array comes in play which starts from 0, if the element is 0 then element is swapped, low is incremented coz now its sorted and mid is incremented to move onto next value.
- if element is 1, there is no need for swapping, mid is incremented and loop moves on to next value.
- If element is 2, element is swapped with ending side of the loop, and high is deremented